projects/service-idler/pipeline.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: v1
+kind: List
+items:
+- kind: ImageStream
+  apiVersion: v1
+  metadata:
+    name: service-idler
+- kind: BuildConfig
+  apiVersion: v1
+  metadata:
+    name: service-idler-binary
+  spec:
+    successfulBuildsHistoryLimit: 1
+    failedBuildsHistoryLimit: 2
+    output:
+      to:
+        kind: ImageStreamTag
+        name: service-idler:binaries
+    source:
+      git:
+        uri: https://github.com/openshift/service-idler.git
+      type: Git
+      dockerfile: |
+				FROM openshift/origin-release:golang-1.10
+				COPY . /go/src/github.com/openshift/service-idler
+				RUN curl -L https://github.com/golang/dep/releases/download/v0.4.1/dep-linux-amd64 -o /go/bin/dep && chmod +x /go/bin/dep
+				RUN cd /go/src/github.com/openshift/service-idler && \
+						/go/bin/dep ensure -vendor-only
+				RUN cd /go/src/github.com/openshift/service-idler && \
+						CGO_ENABLED=0 go build -a -tags netgo -o service-idler-controller ./cmd/controller-manager/main.go
```

Comment:
mixed tabs and spaces.  the highest of computer crimes! :P

Reply:
lol, I'll fix that